.github/workflows/dependency-update.yml

```
@@ -17,40 +30,44 @@ jobs:
+        id: summary
         run: |
-          # Generate updated requirements
-          pip-compile --upgrade --generate-hashes requirements-dev.txt
-          pip-compile --upgrade --generate-hashes requirements-prod.txt
+          echo 'moved<<EOF' >> $GITHUB_OUTPUT
+          git diff --unified=0 -- src/runtime/python/constraints.txt \
+            | grep -E '^[+-][a-zA-Z0-9]' | sort >> $GITHUB_OUTPUT || true
+          echo 'EOF' >> $GITHUB_OUTPUT
 
       - name: Create Pull Request
         uses: peter-evans/create-pull-request@v5
```

actionlint (1.7.12)

[error] the runner of "peter-evans/create-pull-request@v5" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)